### PR TITLE
Increasing linting level

### PR DESCRIPTION
**Pull request by Yoric**

### CODE REVIEW

File: src/Mjolnir.ts [hunk 1]
```diff
@@ -153,7 +153,7 @@ export class Mjolnir {
             await logMessage(LogLevel.DEBUG, "Mjolnir@startup", "Loading protected rooms...");
             await this.resyncJoinedRooms(false);
             try {
-                const data = await this.client.getAccountData(PROTECTED_ROOMS_EVENT_TYPE);
+                const data: Object|null = await this.client.getAccountData(PROTECTED_ROOMS_EVENT_TYPE);
```
Comment (turt2live) [inline src/Mjolnir.ts:156]: this feels a bit overly explicit - the return type should be clear enough?

**Review by turt2live — APPROVED** [review]
The rules feel a bit overly strict, but that's also largely not my problem :D

File: src/Mjolnir.ts (hunk 1 above)
Comment (Yoric) [reply src/Mjolnir.ts:156]: Not in my tests.

**Comment by Yoric** [comment]
Unfortunately, the tests fail. I don't think it's related to my patch.
Do you understand the issue, @turt2live ?

**Pull request merged**